types/node/globals.d.ts

```diff
@@ -142,34 +142,8 @@ interface ErrorConstructor {
     stackTraceLimit: number;
 }
 
-// compat for TypeScript 1.8 and default es5 target
-// if you use with --target es3 or --target es5 and use below definitions,
-// use the lib.es6.d.ts that is bundled with TypeScript 1.8.
-interface MapConstructor { }
-interface WeakMapConstructor { }
-interface SetConstructor { }
-interface WeakSetConstructor { }
-
-interface Set<T> {}
-interface ReadonlySet<T> {}
-
-// Forward-declare needed types from lib.es2015.d.ts (in case users are using `--lib es5`)
-interface Iterable<T> { }
-interface Iterator<T> {
-    next(value?: any): IteratorResult<T>;
-}
-interface IteratorResult<T> { }
-interface IterableIterator<T> { }
-interface AsyncIterableIterator<T> {}
 interface SymbolConstructor {
     readonly observable: symbol;
```

Comment:
I'd have to look to see when this was added, but this value does not actually exist in NodeJS and most likely should also be removed.

Reply:
see #15093, #26034 (observable) and #25269 (asyncIterator)
Not sure if the rxjs (v6) compatibility is still needed.